### PR TITLE
Adding launcher

### PR DESCRIPTION
**Pull request by varun10p**
Adding herbie-style launcher for fpbench. Treating main as library although not sure it is even necessary now.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@ FILTER = racket infra/filter.rkt
 
 setup:
 	raco pkg install --skip-installed -n fpbench --auto generic-flonum
-	raco make main.rkt export.rkt transform.rkt toolserver.rkt evaluate.rkt 
+	raco make fpbench.rkt export.rkt transform.rkt toolserver.rkt evaluate.rkt
```
Comment (bksaiki) [inline Makefile:9]: I think main.rkt and fpbench.rkt should be built?

File: info.rkt [hunk 2]
```diff
@@ -1,6 +1,10 @@
 #lang info
 (define name "fpbench")
 (define version "2.0.3")
+
+(define racket-launcher-names '("fpbench"))
+(define racket-launcher-libraries '("main.rkt"))
```
Comment (bksaiki) [inline info.rkt:6]: I'm think it should be `fpbench.rkt`

Comment (varun10p) [reply info.rkt:6]: Does racket-launcher-libraries not refer to the library file?

File: fpbench.rkt [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+ "src/core2vivado.rkt"
+ "src/core2wls.rkt"
+ "src/fpcore-extra.rkt"
+ "src/fpcore-checker.rkt"
+ "src/fpcore-interpreter.rkt"
+ "src/fpcore-reader.rkt"
+ "src/imperative.rkt"
+ "src/multi-command-line.rkt"
+ "src/range-analysis.rkt"
+ "src/supported.rkt"
```
[the remark this answers is not in the document]
Comment (varun10p) [reply fpbench.rkt:41]: Will do

File: Makefile (hunk 1 above)
Comment (varun10p) [reply Makefile:9]: Will do

File: info.rkt (hunk 2 above)
Comment (bksaiki) [reply info.rkt]: I think it refers to the list of files that get run when the launcher is invoked

**Review by bksaiki — APPROVED** [review]
Looks good!

File: info.rkt (hunk 2 above)
Comment (varun10p) [reply info.rkt]: Yep fixed

**Pull request merged**